scss/_local-navigation.scss

```diff
@@ -0,0 +1,131 @@
+.local-nav {
+  // scss-docs-start local-nav-css-vars
+  --#{$prefix}local-nav-padding-y: #{$local-nav-padding-y};
+  --#{$prefix}local-nav-color: #{$local-nav-color};
+  --#{$prefix}local-nav-bg: #{$local-nav-bg};
+  --#{$prefix}local-nav-hover-color: #{$local-nav-hover-color};
+  --#{$prefix}local-nav-hover-bg: #{$local-nav-hover-bg};
+  --#{$prefix}local-nav-active-color: #{$local-nav-active-color};
+  --#{$prefix}local-nav-active-bg: #{$local-nav-active-bg};
+  --#{$prefix}local-nav-active-marker-width: #{$local-nav-active-marker-width};
+  --#{$prefix}local-nav-border-color: #{$local-nav-border-color};
+  --#{$prefix}local-nav-border-width: #{$local-nav-border-width};
+  // scss-docs-end local-nav-css-vars
+
+  background-color: var(--#{$prefix}local-nav-bg);
+  border-bottom: var(--#{$prefix}local-nav-border-color) solid var(--#{$prefix}local-nav-border-width);
+
+  .local-nav-button {
+    display: block;
+    width: 100%;
+    padding: subtract(var(--#{$prefix}local-nav-padding-y), var(--#{$prefix}local-nav-border-width)) 0 var(--#{$prefix}local-nav-padding-y);
+    font-weight: $font-weight-bold;
+    color: var(--#{$prefix}local-nav-hover-color);
+    background-color: transparent;
+    border: 0;
+
+    > [class*="container"] {
```

Comment:
Using `[class*="container"]` here and `.container-xxl` line 56 isn't an issue? One offers freedom and the other one forces the use of `.container-xxl`.

Reply:
Ah yes, good catch!